Python/ceval.c

```diff
@@ -2973,69 +2612,6 @@ PyUnstable_Eval_RequestCodeExtraIndex(freefunc free)
     return new_index;
 }
 
-static void
-dtrace_function_entry(_PyInterpreterFrame *frame)
```

Comment:
We don't have DTRACE tests and there are a few open issues about it not working as expected.
Rather than get it wrong, I think it is better to not support it for now.
I think @ambv wants to fix up DTRACE support, so let's leave the `pydtrace.h` and the macros for now.

Reply:
Is it worth opening an issue to determine the fate of DTRACE support? @ambv